octree.h

```diff
@@ -2188,7 +2198,8 @@ namespace OrthoTree
     }
 
   public: // Edit functions
-    bool InsertWithRebalancing(std::size_t newEntityID, TVector const& newPoint, std::span<TVector const> const& points) noexcept
+    template<AccessorConcept TAccessor = std::span<const TVector>>
```

Comment:
Maybe it is unavoidable, but I do not want templating the main functions. 
* One instance's main functions could be invoked with totally different `TAccessor` container types.
* With some compilers, it could easily lead to a massive boilerplate with the `template` keyword at invoking. E.g:  `this->template insertWithRebalancing<....>()` (Currently I did not reproduce, but at that time I was forced to write this form.)

 I had to investigate the struct template parameter supplementation with the `TAccessor` before.